README-ch.md

```diff
@@ -0,0 +1,25 @@
+# Mozilla 研究组
```

Comment:
This is so cool. I don't speak chinese so I can't verify the content, do you have a friend who can proofread and give a 👍 ?